nodes/src/nodes/tool_http_request/IGlobal.py

```
@@ -54,27 +50,15 @@
 class IGlobal(IGlobalBase):
     """Global state for http_request."""
 
-    driver: HttpDriver | None = None
+    enabled_methods: Set[str] = set()
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)